Makefile

```diff
@@ -17,7 +18,7 @@ docs/repro.%: docs/repro.%.txt docs/asciidoc.conf
 	a2x --no-xmllint --asciidoc-opts="-f docs/asciidoc.conf" -d manpage -f manpage -D docs $<
 
 repro: repro.in
-	m4 -DREPRO_CONFIG_DIR=$(CONFDIR)/$(PROGNM) $< >$@
+	m4 -DREPRO_VERSION=$(REPRO_VERSION) -DREPRO_CONFIG_DIR=$(CONFDIR)/$(PROGNM) $< >$@
```

Comment:
Sorry, I forgot `$(TAG)` is defined in the Makefile :smile:  please use `-DREPRO_VERSION=$(TAG)`